src/validation.cpp

```diff
@@ -1387,7 +1387,7 @@ int ApplyTxInUndo(Coin&& undo, CCoinsViewCache& view, const COutPoint& out)
 
 /** Undo the effects of this block (with given index) on the UTXO set represented by coins.
  *  When FAILED is returned, view is left in an indeterminate state. */
-static DisconnectResult DisconnectBlock(const CBlock& block, CValidationState& state, const CBlockIndex* pindex, CCoinsViewCache& view)
+static DisconnectResult DisconnectBlock(const CBlock& block, const CBlockIndex* pindex, CCoinsViewCache& view)
```

Comment:
this belongs to "drop state..." commit

Reply:
https://github.com/dashpay/dash/pull/3043/commits/f66ff6be02da31c551fccbd132ed7f45420eab7e